src/helper.js

```diff
@@ -131,6 +131,13 @@ const escapeRegex = (str) => {
   return str.replace(/[.*+\-?^${}()|[\]\\]/g, '\\$&')
 }
 
+/**
+ * @param {string} str
+ */
+const isLogPath = (str) => {
```

Comment:
hmm this is not accurate as some files might have different extensions like .txt or something similar, in some cases they don't even have extensions at all

Reply:
deleted `isLogPath `

Reply:
`isLogPath` was deleted. Any extensions are now allowed